demo/abogomazov/Task 01/Python03.html

```diff
@@ -0,0 +1,68 @@
+﻿<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
xmlns="http://www.w3.org/1999/xhtml" - it is fro html v4 declaration, could you please remove that